scripts/cli-new.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+const yaml = require('js-yaml'),
+    fs = require("fs");
+const colorRed = '\x1b[31m',
+    colorCyan = '\x1b[36m',
+    colorReset = '\x1b[0m';
+
+module.exports = function (args, cb)
+{
+    if (!checkArgs(args, cb)) {
+        return;
+    }
+
+    let cmd = Object.keys(args)[0],
+        waveId = args[cmd];
+
+    createTest(cmd, waveId, cb);
+}
+
+function createTest(cmd, waveId, cb)
+{
+    // lib/waves/{wave id} folder must be empty
+    let waveFolder = 'lib/waves/' + waveId,
+        folderExist = fs.existsSync(waveFolder);
+    if (folderExist && fs.readdirSync(waveFolder).length) {
+        setTimeout(function ()
+        {
+            console.warn(`${colorRed}%s${colorReset}`, `Wave id [${waveId}] already exists.`);
```

Comment:
@dapperdrop @kingo55 
I can scan each config.yml file to make sure there are no clashing wave ids, and checking clashing wave ids in gulp compiler is also necessary in case the user changing the wave id manully.

Reply:
Cool - that makes sense as a change. Always good to check for this stuff so we don't get any clashing during setup or compiling.